util/embeddings/bedrock.py

```
@@ -0,0 +1,172 @@
+                body=json.dumps({"inputText": text}),
+            )
+            result = json.loads(response["body"].read())
+            embedding = result["embedding"]
+
+            if generation:
+                generation.end(
+                    output={"embedding_dimensions": len(embedding)},
+                    usage={"input": len(text)},
+                )
+
+            return embedding
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)